src/utils/daos_autotest.c

```diff
@@ -1000,13 +1023,33 @@ pool_autotest_hdlr(struct cmd_args_s *ap)
 
 	autotest_ap = ap;
 
+	if (autotest_ap->deadline_limit)
+		deadline_limit = autotest_ap->deadline_limit * CLOCKS_PER_SEC;
+
 	step_init();
 
 	for (s = steps; s->func != NULL; s++) {
 		if (s->id < resume)
 			continue;
 		step_new(s->id, s->op);
-		rc = (s->func)();
+
+		int	i;
+		bool	found = false;
+		if (ap->skip_big) {
```

Comment:
(style)  Missing a blank line after declarations

Reply:
I think declarations of i and found should be put first in current block to avoid this warning...

Reply:
daos_autotest.c in the latest commit:

		int	i;
		bool	found = false;

		if (ap->skip_big) {

And per the C style guidelines:
"Variable declarations should be kept to the most internal scope, if practical and reasonable, to simplify understanding of where these variables are accessed and modified, and to avoid errors of using/reusing variables that only have meaning if certain branches are used:"

Perhaps a false positive at this point?

Reply:
Well, I am not against this as a very old C programmer and much before all these extraneous and often changing programming rules have started to be defined ... ;-) BTW, I have already given a +1 to your changes !